datadog/client.go

```diff
@@ -52,6 +50,9 @@ type ClientConfig struct {
 	// Set of name prefixes for metrics to be sent as distributions instead of
 	// as histograms.
 	DistributionPrefixes []string
+
+	// Flag indicating whether to send histograms as `d` type or `h` type
```

Comment:
This doc doesn't make clear whether "true" will send the "d" type or the "h" type. It could also be clearer what the "d" type and the "h" type are - maybe link to a doc?

Reply:
yeah, this could be more clear. Updated! Thanks!